rest/data/project.go

```diff
@@ -64,9 +64,28 @@ func (pc *DBProjectConnector) CreateProject(projectRef *model.ProjectRef, u *use
 	if err != nil {
 		return gimlet.ErrorResponse{
 			StatusCode: http.StatusInternalServerError,
-			Message:    errors.Wrapf(err, "project with id '%s' was not inserted", projectRef.Id).Error(),
+			Message:    errors.Wrapf(err, "inserting project '%s'", projectRef.Identifier).Error(),
 		}
 	}
+
+	newProjectVars := model.ProjectVars{
+		Id: projectRef.Id,
+	}
+
+	err = newProjectVars.Insert()
```

Comment:
If we handle the UI issue from the resolver side wouldn't we be able to avoid these additional DB entries?

Reply:
This is the expected state from the backend's perspective, it's not actually a UI issue.

Reply:
Ok, SGTM